process/charter.html

```diff
@@ -485,7 +485,9 @@ <h3>5.2 <a id="review-eval" name="review-eval">Evaluating the
 
 <h3 id="managing-changes">5.3 Managing Changes to Charters</h3>
 
-<p>Very minor changes may be made in place to a charter. If there are only very minor changes to a charter resulting from the review, the (future) Director's decision includes the original charter URI.</p>
+<p>Very minor changes may be made in place to a charter. If there are only very minor changes to a charter resulting from the review, the (future) Director's decision includes the original charter URI and an explanation of the changes and their rationale.</p>
+<p>If substantive changes are proposed in response to charter review, Team must send the charter with proposed changes, a diff, and an explanation of the changes and their rationale to all who reviewed the charter, copying the member-charters-review@w3.org member-archived mailing list, accompanied by a request for response (in no less than one week) if reviewers have concerns or if the changes would alter their reviews. If the work continues or derives from an existing Group or community effort, Team should also send the diff and a public rationale to that Group or community. These communications should note that the Director has not yet approved the charter.</p>
```

Comment:
> request for a response (in no less than one week)

appears to suggest that responses are not permitted within the first week, whereas I think this probably is supposed to say "within a minimum response period no less than one week" or something like that.